allennlp/training/metrics/boolean_accuracy.py

```diff
@@ -38,12 +37,7 @@ def __call__(self,
         # If you actually passed in Variables here instead of Tensors, this will be a huge memory
```

Comment:
This comment is in `unwrap_variables` - you probably don't need it here anymore.